packages/forma-36-react-components/src/components/Button/Button.tsx

```diff
@@ -78,6 +82,13 @@ export const Button = ({
 
   const Element: ElementType = href ? 'a' : 'button';
 
+  const additionalLinkProps = href
```

Comment:
This doesn't seem necessary anymore.

Reply:
Do we want to allow rendering of (technically) invalid HTML? `rel` and `target` should only exist on `<a>` tags

Reply:
That's where the union type would come into play if we want to get serious 🙂 I'd do it with something like

```
type AnchorProps =
  | {
      href?: undefined;
      rel?: never;
      target?: never;
    }
  | {
      href: string;
      rel?: HTMLProps<HTMLAnchorElement>['rel'];
      target?: HTMLProps<HTMLAnchorElement>['target'];
    };
```

Reply:
I recon that's a good idea. I implemented it this way now :+1:

Reply:
Update: I'll need to figure out how to apply this to an interface without converting it to a type or changing the API surface...